kedro_neptune/__init__.py

```diff
@@ -114,6 +118,8 @@ def neptune_commands():
 PROMPT_PROJECT_NAME = """Pass Neptune project name in a WORKSPACE/PROJECT format or press enter if you want to \
 use $NEPTUNE_PROJECT environment variable:""".replace('\n', '')
 
+NeptuneConfig = namedtuple('NeptuneConfig', ['api_token', 'project', 'base_namespace', 'source_files', 'enabled'])
```

Comment:
Why `namedtuple`.
I think that at this point we can move out config file to separated package, it won't mess up our `__init__` file.

Reply:
Done